project-base/app/src/DataFixtures/Demo/ImageDataFixture.php

```diff
@@ -324,18 +324,24 @@ private function processBlogArticleImages(): void
         }
     }
 
-    private function processMainBlogCategoryImage(): void
+    private function processBlogCategoryImages(): void
     {
-        $mainBlogCategoryImageId = 500;
+        $blogCategoryImagesData = [
+            BlogArticleDataFixture::FIRST_DEMO_BLOG_CATEGORY => 500,
+            BlogArticleDataFixture::FIRST_DEMO_BLOG_SUBCATEGORY => 501,
+        ];
 
-        $mainBlogCategory = $this->getReference(BlogArticleDataFixture::FIRST_DEMO_BLOG_CATEGORY, BlogCategory::class);
-        $names = [];
+        foreach ($blogCategoryImagesData as $blogCategoryReferenceName => $imageId) {
+            $mainBlogCategory = $this->getReference($blogCategoryReferenceName, BlogCategory::class);
```

Comment:
nitpick, this should be named just `$blogCategory`, right?

Reply:
right!